index.html

```diff
@@ -754,6 +755,11 @@
   <ol class='algorithm'>
       <li id="step-create-object">Create a new <a>PerformanceResourceTiming</a> object and
       set <a href="#entryType-attribute">entryType</a> to the DOMString <code>resource</code>.
+        <ul>
+            <li>
+                If the fetch is aborted for any reason, immediately go to step 18 (queue the record).

```

Comment:
Actually, I think this should go into step 7. 

"If fetching the resource is aborted for any reason, abort the remaining steps." > If fetch is aborted due to a network error, go to... queue record.